scripts/background.js

```diff
@@ -1,3 +1,12 @@
+chrome.runtime.onInstalled.addListener(({ reason }) => {
```

Comment:
I added this based on this information on [opening a new tab](https://developer.chrome.com/docs/extensions/reference/api/tabs#open_an_extension_page_in_a_new_tab) on the Chrome for Developers website.